tests/smoke.runner.js

```diff
@@ -347,6 +347,45 @@ const jasmineSpecFiltering = async () => {
     assert.strictEqual(skippedSpecs, 2)
 }
 
+/**
+ * Mocha with spec grouping feature enabled
+ */
+const mochaSpecGrouping = async () => {
+    const { skippedSpecs } = await launch(
+        path.resolve(__dirname, 'helpers', 'config.js'),
+        {
+            specs: [
+                [
+                    path.resolve(__dirname, 'mocha', 'test-empty.js'),
+                    path.resolve(__dirname, 'mocha', 'test-skipped.js'),
+                    path.resolve(__dirname, 'mocha', 'test-skipped-grep.js')
+                ]
+            ]
+        })
+    // Specs will be treated as a group, so no specs will be skipped
```

Comment:
Why are specs in group skipped?

Reply:
Specs in group will **not** be skipped.
```assert.strictEqual(skippedSpecs, 0)```

Reply:
Why is that? If I have my config defined as follows:

```js
specs: [
  "/specA.e2e.js",
  ["/specA.e2e.js"]
]
```

where `specA.e2e.js` has this content:

```js
describe.skip("my feature", () => {
  // ...
})
```

It would skip tests for both files. I think I am missing something here.

Reply:
No, that is right.  In that specific case because "all" the specs in the group would be skipped, then the group would be skipped (no specs to run).  However, in the specific case of the test case, there are specs in the group that will be run and so the group spec will be run and so the returned value of specs skipped will be 0.

Is this desired behaviour? it seems correct to me.

Maybe worth another test case?

Reply:
No you are right, I understand now 👍